src/bot/modules/stats.ts

```diff
@@ -117,4 +117,50 @@ export default class Stats {
 
     return paginate(interaction, embed, pages);
   }
+
+  static async getPoints(member: GuildMember): Promise<number> {
```

Comment:
Tout compte fait, je pense qu'on pourrait donner des points aux corrections qui sont notés comme stale pour la volonté de l'avoir fait, quelque chose comme 3 points ? 🤔

Reply:
Pour ce qui est des approbations des parrains, il te suffit de compter combien de fois l'utilisateur ciblé est dans les approvals & disapprovals.
Je pense que tu peux même le faire à partir de `proposals` directement pas besoin de passer par suggestions & corrections.

Reply:
Pour ce qui est des votes tu peux les récupérer aussi via un include (voir le commentaire qui en parle) il me semble que c'est.